test/methodsSpec.js

```diff
@@ -69,6 +69,14 @@ describe('Methods: ', function() {
 			}, 200);
 		});
 
+		it("ChangeDirection method is working", function(done) {
+			setTimeout(function() {
+				granimInstance.changeDirection('left-right');
+				expect(granimInstance.direction).toEqual('left-right');
+				done();
+			}, 200);
+		});
+
```

Comment:
Can you add a new test, to verify if changing to a custom direction with valid `customDirection` inputs works:
```js
it("ChangeDirection method to a custom direction with valid 'customDirection' inputs is working", function(done) {
	setTimeout(function() {
		granimInstance.customDirection = {
			x0: '20%',
			y0: '200px',
			x1: '30%',
			y1: '400px'
		};
		granimInstance.changeDirection('custom');
		expect(granimInstance.direction).toEqual('custom');
		done();
	}, 200);
});
```